src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -35,6 +38,16 @@ import { ILogService } from 'vs/platform/log/common/log';
 		</div>
 		<div [style.display]="getEditableDisplay()" #editableDropDown style="width: 100%;"></div>
 		<div [style.display]="getNotEditableDisplay()" #dropDown style="width: 100%;"></div>
+		<label #errorMessage tabindex="-1"
+		aria-live="polite" [attr.id]="errorId" aria-atomic="true"
+		*ngIf="!_valid && validationErrorMessages && validationErrorMessages.length!==0 && !isInitState">
+			<ng-container *ngFor="let error of validationErrorMessages">
+				<div  class="dropdown-error-container">
+					<div class="sql codicon error dropdown-error-icon"></div>
```

Comment:
What is the sql class for? I don't recall seeing rules for icons that specify anything about a sql class - if you remove it what happens?

Reply:
Without sql class
![image](https://user-images.githubusercontent.com/6816294/132897425-362add02-5371-4ccc-b2c0-f64a4b6e53a5.png)
With sql class
![image](https://user-images.githubusercontent.com/6816294/132897502-72ace510-1224-47be-bb64-dd6e5915ea90.png)

Reply:
Interesting, can you find out where the rule that defines the icon for the sql one is coming from? The inspector should tell you what css file it's in and what the rule selector is.

Reply:
I got the sql class form the common-icons 

```
.vs .sql.codicon.error,
.vs-dark .sql.codicon.error,
.hc-black .sql.codicon.error {
	content: url("status_error.svg");
}
```

Reply:
There we go, thanks. Guess we kept that since the VS Code one isn't filled in so we had our own custom (would be nice to have named it liked error-filled or something..but ya'know...)